cmd/validate-krew-manifest/main_test.go

```diff
@@ -183,3 +183,58 @@ func Test_isOverlappingPlatformSelectors_overlap(t *testing.T) {
 		t.Fatal("expected overlap")
 	}
 }
+
+func Test_validateLineLength(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected bool
+	}{
+		{
+			name:     "single line below limit",
+			input:    "short string",
+			expected: true,
+		},
+		{
+			name:     "single line above limit",
+			input:    "long string long string long string",
+			expected: false,
+		},
+		{
+			name: "multiple lines, one too long",
+			input: `
+line below limit
+line below limit
+long string long string long string
+line below limit
+line below limit
+`,
+			expected: false,
+		},
+		{
+			name: "long line with a URL",
+			input: "	https://krew.dev/is_really_awesome   ",
+			expected: true,
+		},
+		{
+			name: "multiple lines with long line containing a URL",
+			input: `
+line below limit
+line below limit
+This is a long line with a URL https://krew.dev/is_really_awesome
```

Comment:
`long url ..... some short words`

this test would be beneficial since last fields actually should be on their own line.

Reply:
If I understand correctly, you mean require that the URL is the last word in the line exceeding the allowed line length limit? I think we should not overdo things. Whatever logic we choose needs to be simple and comprehensible.

Reply:
@ferhatelmas Would you be ok with the current logic? If we find it insufficient we can still reiterate.

Reply:
I am good with current implementation. No need to be blocked.

My suggestion for an improvement.

For example, `long-url and some text`, even if it is a long line and could be split like `long-url` and `and some text`, it will pass validation due to url.

Reply:
Should we create an issue to keep track of it or is it a minor?

Reply:
So I think we can gather some experience with the current logic and revisit later. But if you are convinced that we should revisit this at some point, there's no harm in opening an issue right now.